vm/ubpf_vm.c

```diff
@@ -578,22 +583,64 @@ ubpf_run(struct ubpf_vm *vm, void *mem, size_t mem_len, bool jit)
 {
     uint64_t ret;
     char *errmsg;
+    int cntrfd = 0;
+    ubpf_jit_fn fn;
+    struct ubpf_inst_cnt *inst_cnt = vm->inst_cnt;
```

Comment:
Why not just use vm->inst_cnt directly? I don't see the need for a local variable that saves three chars.